CHANGELOG.md

```diff
@@ -1,3 +1,64 @@
+# 1.8.1 (May 2025)
+
+_Changes since 1.8.0_
+
+## Features
+
+### iOS
+
+- Compose resources are being copied into XCFrameworks now [#5292](https://github.com/JetBrains/compose-multiplatform/pull/5292)
+
+### Resources
+
+- Now a compose library with resources may be built and used as XCFramework (it requires Kotlin Gradle plugin 2.2 or higher) [#5294](https://github.com/JetBrains/compose-multiplatform/pull/5294)
+- Gradle Plugin DSL to change the generated Res class name [#5296](https://github.com/JetBrains/compose-multiplatform/pull/5296)
+
+## Fixes
+
+### Multiple Platforms
+
+- Fix incorrect pointer position calculation with rotation around unspecified pivot [#2082](https://github.com/JetBrains/compose-multiplatform-core/pull/2082)
+
+### iOS
+
+- Fix dialogs after modal view controller presentation [#2085](https://github.com/JetBrains/compose-multiplatform-core/pull/2085)
+- Fix issue where `androidx.compose.material3.ModalBottomSheet` closes after any tap [#2086](https://github.com/JetBrains/compose-multiplatform-core/pull/2086)
+- Fix context menu appearance after triple-tap [#2087](https://github.com/JetBrains/compose-multiplatform-core/pull/2087)
+- Fix a memory leak in `ComposeUIViewController` when text input starts [#2088](https://github.com/JetBrains/compose-multiplatform-core/pull/2088)
+- Use the cross-fade animation effect when rotating the screen with interop views [#2101](https://github.com/JetBrains/compose-multiplatform-core/pull/2101)
+- Show an error message when `UIKitViewController` inside `Popup` or `Dialog` [#2102](https://github.com/JetBrains/compose-multiplatform-core/pull/2102)
+- Fix an issue where the keyboard would appear after the second tap when the text input session was intercepted [#2103](https://github.com/JetBrains/compose-multiplatform-core/pull/2103)
+
+### Desktop
+
+- [Linux] Fix `svgPainter` doesn't show any images [#2096](https://github.com/JetBrains/compose-multiplatform-core/pull/2096)
```

Comment:
@igordmn should we mention that this problem happens only with non-english locales in the system due to separator parsing?

Reply:
I have English locale on my Linux and had this issue. Maybe it was UK/US difference, not sure.